### ISSUE TITLE
Disabling URL status check tests

### ISSUE THREAD

**Issue by sigmafelix** [Spatiotemporal-Exposures-and-Toxicology/amadeus#20]
We all found URL status check tests for MODIS datasets take too long to get test results. They eventually failed in most of the cases. I think URL tests, at least for MODIS datasets, need to be disabled. In my opinion, the purpose of URL checking is to make sure that the file is available for downloading. I realized that these tests are testing the server status rather than the functions. In this regard, I suggest disabling such tests from our test suites. We might consider reactivating the tests in the future or add a short vignette to inform users to check data availability with URL status checks.

- [ ] Add `# nocov ` flags in relevant lines for URL status checks
- [ ] Comment out the `check_urls` parts from `test-download_functions.R`

**Comment by mitchellmanware** [comment]
I agree. I can implement the nocov flags and commented code on my branch as I am already dealing with these tests.
